phpunit/class-wp-theme-json-resolver-test.php

```diff
@@ -118,8 +117,10 @@ function test_translations_are_applied() {
 		$this->assertSame( wp_get_theme()->get( 'TextDomain' ), 'fse' );
 		$this->assertSame(
 			array(
-				'color'  => array(
-					'palette' => array(
+				'color'      => array(
+					'custom'         => false,
```

Comment:
These tests now get the defaults for `color.custom`, `color.customGradients`, `typography.customFontSize`, `typography.customLineHeight`, `spacing.units`, and `spacing.customPadding`.

The reason is that the (legacy) theme supports are now pulled from the `WP_Theme_JSON_Resolver_Gutenberg::get_theme_data()` method directly, while, before, these were passed via an argument (and it was empty for the tests).